app/services/data_migrations/backfill_enic_reference_data.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrations
+  class BackfillEnicReferenceData
+    TIMESTAMP = 20210315122316
+    MANUAL_RUN = false
```

Comment:
If this is run automatically after db:migrate, there's a non-zero chance that some records remain where naric_reference is present but enic_reference is nil. Not a huge deal, just worth making a note to check and running this service again manually if so.

Reply:
Yep agree @malcolmbaig - will check the data once deployed